### PR TITLE
Fixed #25526 -- Documented how to output colored text in custom management commands.

### PR DESCRIPTION
**Pull request by elenaoat**
Added short description for CommandBase.style attribute. Also added 2 examples of how the style can be used when outputting to stdout.

### CODE REVIEW

File: docs/howto/custom-management-commands.txt [hunk 1]
```diff
@@ -256,6 +256,12 @@ All attributes can be set in your derived class and can be used in
     because attempting to set the locale needs access to settings. This
     condition will generate a :class:`CommandError`.
 
+.. attribute:: BaseCommand.style
+
+    Instance attribute. Defines the style of the output,
```
Comment (timgraham) [inline docs/howto/custom-management-commands.txt]: We could use complete sentences: "An instance attribute that defines...".
Please wrap the text closer to 79 characters. 
We could also add some more links:
* ```:djadminopt:`--no-color````
* ```:ref:`syntax-coloring```` (to make it clear what the available options are)

Don't forget to link to the pull request on the ticket. Please uncheck "Patch needs improvement" there after you update the pull request. Thanks!

**Comment by elenaoat** [comment]
Added changes specified in the review.

File: docs/howto/custom-management-commands.txt [hunk 2]
```diff
@@ -256,6 +256,15 @@ All attributes can be set in your derived class and can be used in
     because attempting to set the locale needs access to settings. This
     condition will generate a :class:`CommandError`.
 
+.. attribute:: BaseCommand.style
+
+    An instance attribute that defines the style of the output, when writing to,
+    e.g., stdout.
+
+    Used for emphasizing messages by adding a color to them. To avoid text
+    higlighting, pass the :djadminopt:`--no-color` option when running your
```
Comment (timgraham) [inline docs/howto/custom-management-commands.txt]: highlighting

File: docs/howto/custom-management-commands.txt [hunk 3]
```diff
@@ -62,12 +62,12 @@ look like this::
...
                     poll = Poll.objects.get(pk=poll_id)
                 except Poll.DoesNotExist:
-                    raise CommandError('Poll "%s" does not exist' % poll_id)
+                    raise CommandError(self.style.ERROR, 'Poll "%s" does not exist' % poll_id)
 
                 poll.opened = False
                 poll.save()
 
-                self.stdout.write('Successfully closed poll "%s"' % poll_id)
+                self.stdout.write(self.style.SUCCESS, 'Successfully closed poll "%s"' % poll_id)
```
Comment (timgraham) [inline docs/howto/custom-management-commands.txt]: Looks like the syntax is actually `self.style.SUCCESS('Successfully closed poll "%s"' % poll_id)`

File: docs/howto/custom-management-commands.txt [hunk 4]
```diff
@@ -62,12 +62,12 @@ look like this::
                 try:
                     poll = Poll.objects.get(pk=poll_id)
                 except Poll.DoesNotExist:
-                    raise CommandError('Poll "%s" does not exist' % poll_id)
+                    raise CommandError(self.style.ERROR, 'Poll "%s" does not exist' % poll_id)
```
Comment (timgraham) [inline docs/howto/custom-management-commands.txt]: Any `CommandError` text will already display with error highlighting I believe.

File: docs/howto/custom-management-commands.txt [hunk 5]
```diff
@@ -256,6 +256,15 @@ All attributes can be set in your derived class and can be used in
...
     condition will generate a :class:`CommandError`.
 
+.. attribute:: BaseCommand.style
+
+    An instance attribute that defines the style of the output, when writing to,
+    e.g., stdout.
+
+    Used for emphasizing messages by adding a color to them. To avoid text
+    higlighting, pass the :djadminopt:`--no-color` option when running your
+    command. To modify the color palette, see the :ref:`syntax-coloring`.
```
Comment (timgraham) [inline docs/howto/custom-management-commands.txt]: We can also see the list of available style options in the doc which I think would be good to mention.

File: docs/howto/custom-management-commands.txt [hunk 6]
```diff
@@ -256,6 +256,15 @@ All attributes can be set in your derived class and can be used in
     because attempting to set the locale needs access to settings. This
     condition will generate a :class:`CommandError`.
 
+.. attribute:: BaseCommand.style
+
+    An instance attribute that defines the style of the output, when writing to,
+    e.g., stdout.
```
Comment (timgraham) [inline docs/howto/custom-management-commands.txt]: Maybe we should just say "stdout or stderr" (not sure there are any other places it's useful?)
I think some code examples as simple as:
`self.stdout.write(self.style.SUCCESS('...')`
would be great!

**Comment by elenaoat** [comment]
Added changes related to latest comments.

File: docs/howto/custom-management-commands.txt (hunk 5 above)
Comment (elenaoat) [inline docs/howto/custom-management-commands.txt]: I am not sure I understood this comment. In fact, I've tried to find some references in the docs about options. I've found options in the code though, in utils/termcolors.py 
There all available options are provided (SUCCESS, WARNING, NOTICE, etc.).

**Comment by timgraham** [comment]
merged in d26207d56cab21e3767121d53ebaddbbfc78a4ce, thanks!

**Pull request closed without merging**